core/src/main/java/com/linecorp/armeria/server/http/metric/PrometheusExporterHttpService.java

```diff
@@ -36,23 +37,23 @@
 public class PrometheusExporterHttpService extends AbstractHttpService {
     private static final MediaType CONTENT_TYPE_004 = MediaType.parse(TextFormat.CONTENT_TYPE_004);
 
-    private final CollectorRegistry prometheusRegistry;
+    private final CollectorRegistry collectorRegistry;
 
     /**
      * Create a {@link PrometheusExporterHttpService} instance.
-     * @param prometheusRegistry Prometheus registry
+     * @param collectorRegistry Prometheus registry
      */
-    public PrometheusExporterHttpService(CollectorRegistry prometheusRegistry) {
-        Objects.requireNonNull(prometheusRegistry, "prometheusRegistry");
-        this.prometheusRegistry = prometheusRegistry;
+    public PrometheusExporterHttpService(CollectorRegistry collectorRegistry) {
+        requireNonNull(collectorRegistry, "collectorRegistry");
+        this.collectorRegistry = collectorRegistry;
     }
 
     @Override
     protected void doGet(ServiceRequestContext ctx, HttpRequest req,
                          HttpResponseWriter res) throws Exception {
         ByteArrayOutputStream stream = new ByteArrayOutputStream();
         try (OutputStreamWriter writer = new OutputStreamWriter(stream)) {
-            TextFormat.write004(writer, prometheusRegistry.metricFamilySamples());
+            TextFormat.write004(writer, collectorRegistry.metricFamilySamples());
```

Comment:
I think this line is ok. Now that I remember I think it was a missing ``HELP`` and ``TYPE`` messages, when I tried writing my own formatter before. apparently they are used as delimiters between metrics, down stream.

```
# HELP executors_repetitively Generated from dropwizard metric import
# TYPE executors_repetitively counter
executors_repetitively{name="redis-cache-check-timeout-thread.scheduled",} 0.0
# HELP executors_overrun Generated from dropwizard metric import
# TYPE executors_overrun counter
executors_overrun{name="redis-cache-check-timeout-thread.scheduled",} 0.0
# HELP armeria_client_circuit_breaker_counter_count_success Generated from dropwizard metric import
# TYPE armeria_client_circuit_breaker_counter_count_success gauge
armeria_client_circuit_breaker_counter_count_success{handler="PersonaApi",} 0.0
armeria_client_circuit_breaker_counter_count_success{handler="ShopLoggingService",} 1.0
# HELP armeria_client_circuit_breaker_counter_count_failure Generated from dropwizard metric import
# TYPE armeria_client_circuit_breaker_counter_count_failure gauge
armeria_client_circuit_breaker_counter_count_failure{handler="PersonaApi",} 0.0
armeria_client_circuit_breaker_counter_count_failure{handler="ShopLoggingService",} 0.0
# HELP cache_evictions Generated from dropwizard metric import
# TYPE cache_evictions counter
cache_evictions{local="true",name="Stickershop.getRecommendationsForPackage",} 79.0
cache_evictions{local="true",name="Stickershop.getRecommendationsForUser",} 125.0
cache_evictions{local="true",name="authorStorage.getAuthor",} 320.0
cache_evictions{local="true",name="editorsPickService.getAvailableEditorsPick",} 1.0
cache_evictions{local="true",name="editorsPickService.getAvailableEditorsPicks",} 75.0
cache_evictions{local="true",name="personaAPIService.getUserProfile",} 612.0
cache_evictions{local="true",name="productStorage.getProduct",} 0.0
cache_evictions{local="true",name="productStorage.getProductByVersion",} 690.0
cache_evictions{local="true",name="recommendationDataStorage.getRecommendationForProductDataList",} 32.0
cache_evictions{local="true",name="recommendationMetaStorage.getRecommendationMeta",} 19.0
```
But you should be ok with just using ``TextFormat.write004``